edb/tools/test/decorators.py

```diff
@@ -25,12 +25,13 @@
 skip = unittest.skip
 
 
-def xfail(reason, *, unless=False):
+def xfail(reason, *, unless=False, allow_failure=False):
```

Comment:
Perhaps introduce another decorator for better visibility, e.g. `xerror`?

Reply:
And turn the old xfail into xerror?

Reply:
Yeah.